mod/wikirate_csv_import/set/type_plus_right/source/file/import_answers_from_source.rb

```diff
@@ -0,0 +1,77 @@
+include_set Abstract::ImportWithCompanies
```

Comment:
is this new?

Reply:
it's old but could be that you deleted it during your source clean up